docs/src/app/layout.tsx

```diff
@@ -1,23 +1,26 @@
-import * as React from 'react';
 import { Metadata } from 'next';
-import { GoogleAnalytics } from 'docs/src/components/GoogleAnalytics';
+import * as React from 'react';
```

Comment:
We have a convention to import React first.

Reply:
Can I add an ES lint rule to do import order so that it's fixed automatically then? I agree with the convention, I did similar for Radix Themes with a couple other ordering rules in the mix, just don't think a human should waste time doing it manually

Reply:
sure!